test/sol6/kyberNetwork.js

```diff
@@ -1385,6 +1385,17 @@ contract('KyberNetwork', function(accounts) {
                         let txResult = await network.tradeWithHintAndFee(network.address, srcToken.address, srcQty, ethAddress, taker,
                             maxDestAmt, minConversionRate, platformWallet, platformFeeBps, hint);
                         console.log(`token -> ETH (${tradeStr[hintType]}): ${txResult.receipt.gasUsed} gas used`);
```

Comment:
its good to have all these tests.
but it should be PRed to Anton's branch with the modified events.

Reply:
removed